packages/flutter_tools/lib/src/build_runner/devfs_web.dart

```diff
@@ -382,7 +421,10 @@ class WebAssetServer implements AssetReader {
   }
 
   @override
-  Future<String> dartSourceContents(String serverPath) {
+  Future<String> dartSourceContents(String serverPath) async {
+    if (serverPath == null || serverPath.trim().isEmpty) {
```

Comment:
This is an interesting guard. Not that it's wrong by does `package:dwds` ever request null paths?

Reply:
I noticed that devtools was requesting null paths before the change to the tracked library names. I will re-verify this

Reply:
Updated - we only get a null path when devtools tries to request the web-entrypoint file with the org-dartlang-app scheme. I made this check more specific, but we should probably fix/hide this file otherwise